lib/Swagger/src/Discussion/Api/ContributionApi.php

```diff
@@ -0,0 +1,348 @@
+<?php
```

Comment:
I think the preference has been to use swagger client. I did ask a few in the team here about whether I should just add the partial API client, but we decided that would be suboptimal for future usage by other teams such as CAKE.